src/backend/localify.rs

```diff
@@ -58,7 +58,7 @@ impl<'a, V: Visitor> BlockVisitor<'a, V> {
     fn new(body: &'a FunctionBody, trees: &'a Trees, visitor: V) -> Self {
         log::trace!(
             "localify: running on:\n{}",
-            body.display_verbose("| ", None)
+            body.display_verbose("| ", None, &crate::NOPPrintDecorator::default())
```

Comment:
Rather than change the existing signature and require this `Default` impl, can we add a new method on the `FunctionBody`, maybe something like `FunctionBody::display_with_decorator()`?

Reply:
Yes, I can add a new display function instead.